src/accessors-geog.cpp

```diff
@@ -12,23 +14,20 @@ const double EARTH_RADIUS_METERS = 6371.01 * 1000;
 PyObjectGeography centroid(PyObjectGeography a) {
     const auto& a_ptr = a.as_geog_ptr()->geog();
     auto s2_point = s2geog::s2_centroid(a_ptr);
-    std::unique_ptr<Point> point =
-        make_geography<s2geog::PointGeography, spherely::Point>(s2_point);
-    return PyObjectGeography::from_geog(std::move(point));
+    return make_py_geography<s2geog::PointGeography>(s2_point);
 }
 
 PyObjectGeography boundary(PyObjectGeography a) {
     const auto& a_ptr = a.as_geog_ptr()->geog();
     auto s2_obj = s2geog::s2_boundary(a_ptr);
-    // TODO return specific subclass
     auto geog_ptr = std::make_unique<spherely::Geography>(std::move(s2_obj));
     return PyObjectGeography::from_geog(std::move(geog_ptr));
```

Comment:
Above you made a shortcut for creating a PyObjectGeography from an s2 object (`make_py_geography`), should we do something similar for this case creating it from a s2geography object? (the name `s2_obj` here is a bit confusing, because it is a `s2geography::Geography`, not some `s2` object)

Eg add a `PyObjectGeography::from_s2geog()` to do this in one line? (this is probably not strictly related to this PR, so can also do that separately, this PR is already big)

Reply:
Yes either option is fine by me.

Reply:
I went for the latter option (`make_py_geography` overload).